addons/dogtags/functions/fnc_inventoryDisplayLoad.sqf

```diff
@@ -0,0 +1,50 @@
+#include "..\script_component.hpp"
+/*
+ * Author: johnb43
+ * Executed every time an inventory display is opened.
+ *
+ * Arguments:
+ * 0: Inventory display <DISPLAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [DISPLAY] call ace_dogtags_fnc_inventoryDisplayLoad
+ *
+ * Public: No
+ */
+
+disableSerialization;
+
+params ["_display"];
+
+// Monitor changes that can happen and force our update
+private _dummyControl = _display ctrlCreate ["RscMapControl", -1];
+
+_dummyControl ctrlSetPosition [0, 0, 0, 0];
+_dummyControl ctrlCommit 0;
+
+_dummyControl ctrlAddEventHandler ["Draw", {
+    private _display = ctrlParent (_this select 0);
```

Comment:
could use mouseEH (part 1/2)
```suggestion
// Monitor changes that can happen and force our update
private _update = {
    params ["_display"];
```

Reply:
What's the benefit of the mouse EH?
Wouldn't a PFH be better than that (https://github.com/IDI-Systems/acre2/pull/1249)?
But even then - why go with a PFH?

I went with the Draw EH because the inventory filter uses it.